src/panels/lovelace/views/hui-sections-view.ts

```diff
@@ -348,7 +346,8 @@ export class SectionsView extends LitElement implements LovelaceViewElement {
 
       .section-actions {
         position: absolute;
-        top: 0;
+        height: 36px;
+        top: calc(var(--row-height) + var(--row-gap) - 36px + 2px);
```

Comment:
Messing around with these variables did no longer attach the buttons to the section. I may not be right in how these can be used, but it may need some rechecking.

Reply:
This should not be an issue because we are using variables when we can (36px and 2px are hardcoded values for the actions height and the border width) so it should work. 
But I agree, it's not clean. I will used to better attach this buttons to the container of the section.